src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -44248,6 +44255,7 @@ static class E
 
 42.M();
 42.M2();
+E.M(42);
```

Comment:
Nit: I know you didn't make this test source in this test, but consider refactoring to just set a preprocessor symbol through options and avoid repeating the same source twice, that we then have to keep in sync.